newIDE/app/src/EventsFunctionsExtensionsLoader/CodeWriters/LocalEventsFunctionCodeWriter.js

```diff
@@ -16,7 +17,9 @@ export const makeLocalEventsFunctionCodeWriter = ({
   onWriteFile,
 }: EventsFunctionCodeWriterCallbacks): EventsFunctionCodeWriter => {
   // The generated code for extensions will be stored in a temporary directory
-  const outputDir = os.tmpdir() + '/GDGeneratedEventsFunctions';
+
+  const outputDir =
+    os.tmpdir() + `/GDGeneratedEventsFunctions-` + getUID();
```

Comment:
```JS
path.join(os.tmpdir(), 'GDGeneratedEventsFunctions-' + getUID());
```
What do you think?

Reply:
Fixed

Reply:
It's format

Reply:
Ask to the auto-formatter.
I don't know why too.

Reply:
Oh ok I understand

Reply:
Well this is working. And this is correct

Reply:
But if you **insist**

Reply:
Finally fixed